frontend/src/pages/MapPage/LayerSidebar.tsx

```diff
@@ -90,7 +91,15 @@ function LayerSidebar({
                   onChange={(_e, {value}) => setMapConfigFlag('obsRoads.maxCount', value)}
                 />
               </List.Item>
-            ) : null}
+              <List.Item>
+                <ColorMapLegend map={_.chunk(colorByCount('obsRoads.maxCount',mapConfig.obsRoads.maxCount, reds).slice(3), 2)} />
```

Comment:
`ReferenceError: colorByCount is not defined`

Reply:
This was a missing commit and is now fixed.